### ISSUE TITLE
Reorganize research data page

### ISSUE THREAD

**Issue by madprime** [OpenHumans/open-humans#327]
Currently connections can be listed twice. If connected, data import results are listed at the top. Then, all potential connections with possible actions (connect or redo import) are listed at the bottom.

Reorganize page to only have a section for each connection once (don't list connected sources on the bottom again). In other words, once a connection is made, present both resulting files and available actions for that connection together on the top.

The research data page is the one located here: https://www.openhumans.org/member/me/research-data/

**Comment by jasonbobe** [comment]
thanks

**Comment by jasonbobe** [comment]
:)

**Comment by BenjaminHCCarr** [comment]
Both the default: https://www.openhumans.org/
And https://www.openhumans.org/research/
Look much improved! Thanks @madprime 
Are you happy with it? Opportunity to close an issue?

**Comment by beaugunderson** [comment]
@BenjaminHCCarr the 'research data page' refers specifically to this page:

https://www.openhumans.org/member/me/research-data/

there's still some work we'd like to do in cleaning that one up :)

**Comment by BenjaminHCCarr** [comment]
@beaugunderson ah okay, yes, that is a little clunky. Just so I don't create another thread why is activities-list a div call \# and not a regular URL:

works: https://www.openhumans.org/#activities-list
doesn't: https://www.openhumans.org/activities-list

the 404 is a little odd for that too:
`<body id="page-activities-list" class="404">`

I didn't think the class="404" could go in a <body> tag, but you're the expert, I thought it had to be a in proper <div>

This odd activities-list page also make linking harder. Like my issue #567 example, the url for circles is:
`<a href="https://www.openhumans.org/#activity-direct-sharing-2">`
where I think it should be ( and what I tried first)
'https://www.openhumans.org/activities-list#activity-direct-sharing-2'

I haven't dealt a ton with node (and obviously you have), but is it not possible to root the dynamic page without the leading \#?

it would make referential links much cleaner.

**Comment by madprime** [comment]
@BenjaminHCCarr

I added the anchor as an afterthought. The goal of #526 was to eliminate the "Activities" tab and simply have that list be the home page. But for the logged out user, there's some elements displayed before the activities list. So I added an anchor to let us have a link that jumps the user to the activities list (in case we wanted to reference it for a user that hasn't joined or isn't logged in).

The id and class for "body" are automatically generated in the functions here: https://github.com/PersonalGenomesOrg/open-humans/blob/master/open_humans/templatetags/utilities.py#L154
https://github.com/PersonalGenomesOrg/open-humans/blob/master/open_humans/templatetags/utilities.py#L167
